plugin/inc/installsteps/crontasksinstallstep.class.php

```diff
@@ -43,10 +43,21 @@ class CronTasksInstallStep
             'mode'      => CronTask::MODE_EXTERNAL,
             'frequency' => 86400,
             'param'     => 0,
+            'state'     => 0,
             'hourmin'   => 5,
             'hourmax'   => 8,
             'comment'   => '',
-        ]
+        ], [
+            'itemtype' => 'PluginIservicePrinterDailyAverageCalculator',
+            'name' => 'PrinterDailyAverageCalculator',
+            'mode' => CronTask::MODE_EXTERNAL,
+            'frequency' => 86400,
+            'param' => 0,
+            'state' => 1,
+            'hourmin' => 0,
+            'hourmax' => 24,
+            'comment' => 'emaintenance@expertline.ro',
```

Comment:
This should run during the night also
```suggestion
            'itemtype' => 'PluginIservicePrinterDailyAverageCalculator',
            'name' => 'PrinterDailyAverageCalculator',
            'mode' => CronTask::MODE_EXTERNAL,
            'frequency' => 86400,
            'param' => 0,
            'state' => 1,
            'hourmin' => 4,
            'hourmax' => 7,
            'comment' => 'emaintenance@expertline.ro',
```